AWS: failed with 'dict object' has no attribute 'tag_openshift_role_master'

TASK [instance-groups : Add masters to requisite groups] ***********************
task path: /root/AWS/openshift-ansible-contrib/reference-architecture/aws-ansible/playbooks/roles/instance-groups/tasks/main.yaml:9
fatal: [localhost]: FAILED! => {
    "failed": true
}

MSG:

'dict object' has no attribute 'tag_openshift_role_master'

Glad we got passed this point. We shall compare notes tomorrow about the
us-east-1 issue. This morning it passed ci in us-east-1 so it may have been
a later in the day aws outage or something with the account. Sometimes
cloudformations will attempt to use a vpc classic subnet which causes
failure

On Jan 2, 2017 4:04 PM, "gt000" <notifications@github.com> wrote:

> Adding us-east-2 to ec2.ini makes the script go forward correctly. It
> would be interesting to understand the reason for the other issue, which
> brought us to substitute us-east-1 to us-east-2.
> The message was "CREATE_FAILED AWS::EC2::Subnet PublicSubnet3 Value
> (us-east-1c) for parameter availabilityZone is invalid. Subnets can
> currently only be created in the following
> availability zones: us-east-1e, us-east-1a, us-east-1b, us-east-1d."
> It seems the script tries to create a subnet in us-east-1c but it seems
> it's forbidden.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-ansible-contrib/issues/143#issuecomment-270020161>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADoslG6xZuhwSORMSw-xWWZ2fRQ8cwSRks5rOWZzgaJpZM4LZJG8>
> .
>

--- a/reference-architecture/aws-ansible/inventory/aws/hosts/ec2.ini
+++ b/reference-architecture/aws-ansible/inventory/aws/hosts/ec2.ini
@@ -11,7 +11,7 @@
 # AWS regions to make calls to. Set this to 'all' to make request to all regions
 # in AWS and merge the results together. Alternatively, set this to a comma
 # separated list of regions. E.g. 'us-east-1,us-west-1,us-west-2'
-regions = us-east-1,us-west-1,us-west-2,sa-east-1,eu-west-1,eu-central-1,ap-northeast-1,ap-southeast-1,ap-southeast-1,ap-southeast-2
+regions = us-east-1,us-east-2,us-west-1,us-west-2,sa-east-1,eu-west-1,eu-central-1,ap-northeast-1,ap-southeast-1,ap-southeast-1,ap-southeast-2
 regions_exclude = us-gov-west-1,cn-north-1
 
 # When generating inventory, Ansible needs to know how to address a server.

--- a/reference-architecture/aws-ansible/inventory/aws/hosts/ec2.ini
+++ b/reference-architecture/aws-ansible/inventory/aws/hosts/ec2.ini
@@ -11,7 +11,7 @@
 # AWS regions to make calls to. Set this to 'all' to make request to all regions
 # in AWS and merge the results together. Alternatively, set this to a comma
 # separated list of regions. E.g. 'us-east-1,us-west-1,us-west-2'
-regions = us-east-1,us-east-2,us-west-1,us-west-2,sa-east-1,eu-west-1,eu-central-1,ap-northeast-1,ap-southeast-1,ap-southeast-1,ap-southeast-2
+regions = all
 regions_exclude = us-gov-west-1,cn-north-1
 
 # When generating inventory, Ansible needs to know how to address a server.

